packages/trace-viewer/src/ui/networkResourceDetails.tsx

```diff
@@ -79,6 +79,12 @@ const RequestTab: React.FunctionComponent<{
   return <div className='network-request-details-tab'>
     <div className='network-request-details-header'>URL</div>
     <div className='network-request-details-url'>{resource.request.url}</div>
+    <div className='network-request-details-header'>Method</div>
```

Comment:
Can we reduce repetition ( Method Method:, Status Status:), perhaps put non header details into one `General` section and make `Request URL:`, `Method:` and `Status:` individual lines under it?

Reply:
All done @yury-s 

<img width="848" alt="2024-06-15_10-07-28" src="https://github.com/microsoft/playwright/assets/50574915/b23a9669-0f3c-499d-8677-04798dc84f28">